src/ape/managers/converters.py

```diff
@@ -83,6 +83,18 @@ def convert(self, value: str) -> AddressType:
         return to_checksum_address(value)
 
 
+class BytesAddressConverter(ConverterAPI):
+    """
+    A converter that converts a raw bytes address to an ``AddressType``.
+    """
+
+    def is_convertible(self, value: Any) -> bool:
+        return isinstance(value, bytes) and len(value) == 20
```

Comment:
Is there a potential unintended conversion here? Like can these conditions be true without it being an address?

Is there any bad that could happen from unintended conversion here?

Reply:
Can update to use `eth_utils.is_canonical_address`:
https://github.com/ethereum/eth-utils/blob/master/eth_utils/address.py#L88

Reply:
On second thought, this is okay. Any random string of 20 bytes is a valid address.

Reply:
since bytes expecting args always expect bytes, the unnecessary conversion I was worried about would not happen